app/src/pages/project/ProjectRetentionPolicyCard.tsx

```diff
@@ -0,0 +1,169 @@
+import React, { useMemo } from "react";
+import {
+  graphql,
+  useFragment,
+  useMutation,
+  useRefetchableFragment,
+} from "react-relay";
+import cronstrue from "cronstrue";
+
+import { Card } from "@arizeai/components";
+
+import { Flex, Link, Text, View } from "@phoenix/components";
+import { ProjectTraceRetentionPolicySelect } from "@phoenix/components/retention/ProjectTraceRetentionPolicySelect";
+import {
+  useNotifyError,
+  useNotifySuccess,
+  useViewerCanManageRetentionPolicy,
+} from "@phoenix/contexts";
+import { createPolicyDeletionSummaryText } from "@phoenix/utils/retentionPolicyUtils";
+
+import { ProjectRetentionPolicyCard_policy$key } from "./__generated__/ProjectRetentionPolicyCard_policy.graphql";
+import { ProjectRetentionPolicyCard_query$key } from "./__generated__/ProjectRetentionPolicyCard_query.graphql";
+import { ProjectRetentionPolicyCardQuery } from "./__generated__/ProjectRetentionPolicyCardQuery.graphql";
+import { ProjectRetentionPolicyCardSetProjectRetentionPolicyMutation } from "./__generated__/ProjectRetentionPolicyCardSetProjectRetentionPolicyMutation.graphql";
+
+export const ProjectRetentionPolicyCard = ({
+  project,
+  query,
+}: {
+  project: ProjectRetentionPolicyCard_policy$key;
+  query: ProjectRetentionPolicyCard_query$key;
+}) => {
+  const canManageRetentionPolicy = useViewerCanManageRetentionPolicy();
+  const notifySuccess = useNotifySuccess();
+  const notifyError = useNotifyError();
+
+  const queryKey = useFragment(
+    graphql`
+      fragment ProjectRetentionPolicyCard_query on Query {
+        ...ProjectTraceRetentionPolicySelectFragment
+      }
+    `,
+    query
+  );
+  const [data] = useRefetchableFragment<
```

Comment:
I don't think this needs to be a refetchable fragment if you aren't using the refetch callback. I looked at the docs and useFragment will also refetch if the fragment it defines is returned by a mutation response.